### ISSUE TITLE
Disable dropout for prediction/evaluation

### ISSUE THREAD

**Issue by mhham** [zalandoresearch/flair#216]
I just trained a simple NER tagger (with only french fasttext word embeddings, and no contextual string embeddings).

The model is the following:
```python
SequenceTagger(
  (embeddings): StackedEmbeddings(
    (list_embedding_0): WordEmbeddings()
  )
  (word_dropout): WordDropout()
  (locked_dropout): LockedDropout()
  (embedding2nn): Linear(in_features=300, out_features=300, bias=True)
  (rnn): LSTM(300, 256, bidirectional=True)
  (linear): Linear(in_features=512, out_features=12, bias=True)
)
```

When predicting the tags of a text, the output changes everytime, which suggests that dropout is still applied at prediction. 
I think it should not be the case, and same thing for evaluation

(I am using version 0.3.2)

**Comment by stefan-it** [comment]
Just pass `0.0` to the dropout method constructors, like `WordDropout(0.0)` and `LockedDropout(0.0)`. If you pass nothing, then the default values (0.5 and 0.05`) are used :)

**Comment by alanakbik** [comment]
Interesting, how are you loading the tagger? It should be put on .`eval()` mode when you load it, which should disable all dropout.

**Comment by mhham** [comment]
I first load it with : 
```python
tagger = SequenceTagger.load_from_file('resources/taggers/example-ner/final-model.pt')
```

Then train it with : 

```python
trainer: SequenceTaggerTrainer = SequenceTaggerTrainer(tagger, corpus)
trainer.train('resources/taggers/example-ner',
              learning_rate=0.1,
              mini_batch_size=128,
              max_epochs=150,
              checkpoint=True)
```

Then predict right after training
```python
tagger.predict(sentences)
```

**Comment by alanakbik** [comment]
Ah I see. After training, it is probably still in training mode (during training, dropout is applied). Try calling `tagger.eval()` after training, which should disable all dropout - then it should always give the same results.

**Comment by mhham** [comment]
Got it ! It might be interesting to keep taggers in eval mode, unless they are trained ?

**Comment by alanakbik** [comment]
yes good point, will add that!

**Comment by tabergma** [comment]
@mhham Any updates?
